packages/devtools_app/lib/src/service/isolate_manager.dart

```diff
@@ -249,14 +249,32 @@ class IsolateManager extends Disposer {
       service.onDebugEvent.listen(_handleDebugEvent),
     );
 
-    // We don't yet known the main isolate.
+    // We don't know the main isolate yet.
     _mainIsolate.value = null;
   }
 
-  Future<Isolate?> getIsolateCached(IsolateRef isolateRef) {
-    final isolateState =
-        _isolateStates.putIfAbsent(isolateRef, () => IsolateState(isolateRef));
-    return isolateState.isolate;
+  /// Resturns isolate by its reference.
+  ///
+  /// The method will return immediately in most cases
```

Comment:
return immediately is misleading as the return value is a future.
This method name has always been bad. Some other ideas:
isolateFor(isolateRef)
isolate(isolateRef)

delete this method completely and just call
isolateState(isolateRef).isolate
everywhere this method was used.

Reply:
deleted